app/src/frontend/ConfirmationModal.css

```diff
@@ -0,0 +1,7 @@
+.modal.modal-show {
```

Comment:
Niggle comment on filename - please stick with the hyphen-case-convention - so `confirmation-modal.css`

Reply:
ok! will correct